src/odoc/bin/main.ml

```diff
@@ -666,7 +694,7 @@ module Depends = struct
       Term.(const list_dependencies $ input)
 
     let info =
-      Term.info "compile-deps"
+      Term.info "compile-deps" ~docs:section_legacy
```

Comment:
compile-deps shouldn't be legacy - it's used in the reference driver.

Reply:
Moved back.